### DISCUSSION TITLE
Confused over stock - consumed by build order.

### DISCUSSION THREAD

**Discussion by StormTrooper** [inventree/InvenTree#5058] [Q&A] [answered]
Have been using Inventree for a few months now and seeing a few confusing things under stock.

On a lot of my parts  I have multiple line items for parts showing consumed by build order.  All of the build orders have been completed so I would think this should dissapear once the build order is complete, but this is not the case. 

The available stock still shows the correct count but am confused as to why I"m seeing all these items under stock. 

![image](https://github.com/inventree/InvenTree/assets/1429043/96e7cc65-9717-49e6-a2cd-ebe54869b01c)

**Comment by SchrodingersGat** [comment, marked as answer]
> All of the build orders have been completed so I would think this should dissapear once the build order is complete, but this is not the case.

This is the intended behaviour, to allow traceability for stock items even after they have been consumed. 

You can simply filter by "available" to hide those items

**Reply by StormTrooper** [reply]
Thanks.  I was playing around with filters so I think previously it was set to avaible and then I changed it. 

Explains why I'm seeing them all now. 

Good to know it's expected behaviour.
